src/include/timestamp_inline.h

```diff
@@ -36,12 +39,16 @@
 /* Check if the stop time window is set. */
 #define WT_TIME_WINDOW_HAS_STOP(tw) ((tw)->stop_txn != WT_TXN_MAX || (tw)->stop_ts != WT_TS_MAX)
 
+/* Check if the prepare time window is set. */
+#define WT_TIME_WINDOW_HAS_PREPARE(tw) \
+    ((tw)->prepare == 1 && ((tw)->prepare_ts != WT_TS_NONE || (tw)->prepared_txn != WT_TXN_NONE))
```

Comment:
We should only use prepared flag to check wether it is a prepared here. We can have a committed prepared transaction with valid prepared ts and prepared id.

Reply:
This macro i want to use to check if the prepared ts and prepared id fields are set, not for checking the prepared flag